include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -867,6 +872,13 @@ SpatialOperatorBase<dim, Number>::get_viscosity() const
   return param.viscosity;
 }
 
+template<int dim, typename Number>
+double
+SpatialOperatorBase<dim, Number>::get_density() const
```

Comment:
I did not need this in the end, should I remove it again?

Reply:
yes, should be removed again.

Reply:
I did :)